inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/InlongClient.java

```diff
@@ -224,6 +224,15 @@ static InlongClient create(String serviceUrl, ClientConfiguration configuration)
      */
     PageResult<ClusterNodeResponse> listNode(ClusterPageRequest request);
 
+    /**
+     * Get DP node info by groupId and protocol
+     *
+     * @param groupId inlong group id
+     * @param protocolType protocol type, such as: TCP,HTTP
+     * @return DP list
+     */
+    List<ClusterNodeResponse> listDPNode(String groupId, String protocolType);
```

Comment:
would it be clearer if we use the full name DataProxy rather than DP?

Reply:
done